src/Factory/FactoryInterface.php

```diff
@@ -23,7 +23,7 @@ interface FactoryInterface
      * Create an object
      *
      * @param  string             $requestedName
-     * @param  null|array         $options
+     * @param  null|array<mixed>  $options
```

Comment:
`array` is an alias of `array<int|string, mixed>`? Why is this needed explicitly? :thinking:

Reply:
Yeah, that's what i was thinking. Maybe i should report to phpstan instead.